pkg/pipelineascode/pipelineascode.go

```diff
@@ -40,14 +57,25 @@ func getRepoByCRD(cs *cli.Clients, url, branch, eventType string) (apipac.Reposi
 
 func Run(cs *cli.Clients, runinfo *webvcs.RunInfo) error {
 	var err error
+	var maintekton TektonYamlConfig
+
 	var ctx = context.Background()
+
 	checkRun, err := cs.GithubClient.CreateCheckRun("in_progress", runinfo)
 	if err != nil {
 		return err
 	}
 	runinfo.CheckRunID = checkRun.ID
 
-	repo, err := getRepoByCRD(cs, runinfo.URL, runinfo.Branch, "pull_request")
+	maintektonyaml, _ := cs.GithubClient.GetFileFromDefaultBranch(filepath.Join(tektonDir, tektonConfigurationFile), runinfo)
```

Comment:
This would always get the pipeline yaml from main/master ?

Reply:
Yep correct or more accurately as defined as the default branch in github.